Added C# code example to addMessagecl.md (#511)
citizenfx/fivem-docs: content/docs/resources/chat/exports/addMessagecl.md

  })
end)
```

##### C# Example:
```c#

> EventHandlers["onResourceStart"] += new Action<string>(resourceName); // this in public ClientMain() {...}

Reviewer: ```suggestion
// add this in the class constructor
EventHandlers["onResourceStart"] += new Action<string>(OnResourceStart);
```

//...

> void resourceName([FromSource] string resourceName)

Reviewer: ```suggestion
void OnResourceStart(string resourceName)
```

There is no source on these events

{

>     if (API.GetCurrentResourceName() != resourceName) { return; }

Reviewer: ```suggestion
    if (API.GetCurrentResourceName() != resourceName) return;
```

    TriggerEvent("chat:addMessage", new
    {
        color = new[] { 255, 0, 0 },
        multiline = true,
        args = new[] { "[SYSTEM]", API.GetCurrentResourceName() + " has started." }
    });

> }

Reviewer: Given that this example is in the exports section, should this not use the export instead of the event?

Author: Yeah you’re right exports would be a good idea, this was a very low quality commit, I just sorta threw this together on the side! I’m currently painfully viewing this on mobile, but from what I can see you added the missing {} and removed the from Source plus added a more helpful comment. 

Once I’m on my laptop I can quickly change this to use Exports!
